### PR TITLE
Ctr 354

### PR DESCRIPTION
**Pull request by Pascoe121**

### CODE REVIEW

**Comment by Pascoe121** [comment]
Dont merge till CAS changes in prod

File: app/views/components/input_text.scala.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
         <span id="hint-line-2" class="form-hint">@hintLine2</span>
         }
         @if(viewModel.errorKey.nonEmpty){
-        <span class="error-notification" id="error-message-@{viewModel.id}-input">@messages(viewModel.errorKey)</span>
+        <span class="error-notification" id="error-message-@{viewModel.id}-input">@messages(viewModel.errorKey, viewModel.value.get)</span>
```
Comment (vinnicombe) [inline app/views/components/input_text.scala.html]: Still a little uncomfortable about this option.get, can we have a good look at option.getOrElse("") to see if there's any negative impact to providing an empty string?

Comment (Pascoe121) [reply app/views/components/input_text.scala.html]: There would be no negative impact, however the get would never be called unless there was a value, as the bind would error on blank before it tried to fill the value i believe. But for confidence ill change it to getOrElse("")

**Comment by Pascoe121** [comment]
Changes made

**Pull request merged**